src/main/java/com/somemore/global/exception/ExceptionMessage.java

```diff
@@ -10,7 +10,7 @@ public enum ExceptionMessage {
 
     NOT_EXISTS_CENTER("존재하지 않는 기관 ID 입니다."),
     NOT_EXISTS_COMMUNITY_BOARD("존재하지 않는 게시글 입니다."),
-    UNAUTHORIZED_COMMUNITY_BOARD("게시글 삭제 권한이 없습니다."),
+    UNAUTHORIZED_COMMUNITY_BOARD("해당 게시글에 권한이 없습니다."),
     NOT_EXISTS_LOCATION("존재하지 않는 위치 ID 입니다."),
```

Comment:
```suggestion
    NOT_FOUND("%s를 찾을 수 없습니다."),
    UNAUTHORIZED("%s에 대한 권한이 없습니다."),
```

이렇게 사용하고 아래에서 메시지 게터에 도메인이나 맥락에 대해서 동적으로 파라미터를 사용하는 방법은 어떨까요?

Reply:
다 같이 얘기해보면 좋을 거 같아요!

Reply:
넵